content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst

```diff
@@ -0,0 +1,132 @@
+-----------
+
+To process a *Pass - Fail* quality check on an order, select a manufacturing order or inventory
+order (receipt, delivery, return, etc.), for which a check is required. Manufacturing orders can be
+selected by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`,
+and clicking on an order. Inventory orders can be selected by navigating to
+:menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation card, and
+selecting an order.
+
+On the selected manufacturing or inventory order, a purple :guilabel:`Quality Checks` button appears
+at the top of the order. Click the button to open the :guilabel:`Quality Check` pop-up window, which
+shows all of the quality checks required for that order.
+
+To process a *Pass - Fail* quality check, follow the instructions shown on the :guilabel:`Quality
+Check` pop-up window. If the criteria for the check is met, click the :guilabel:`Pass` button at the
+bottom of the window. If the criteria is not met, click the :guilabel:`Fail` button.
+
+.. image:: pass_fail_check/pass-fail-check-pop-up.png
+   :align: center
+   :alt: A Pass - Fail quality check pop-up window on a manufacturing or inventory order.
+
+If a quality alert must be created, click the :guilabel:`Quality Alert` button that appears at the
+top of the manufacturing or inventory order after the check fails. Clicking :guilabel:`Quality
+Alert` opens a quality alert form on a new page. For a complete guide on how to fill out quality
+alert forms, view the documentation on :ref:`quality alerts
+<quality/quality_management/quality-alerts>`.
+
+On a work order
+---------------
+
+When configuring a |QCP| that is triggered during manufacturing, a specific work order can also be
+specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a work order is
+specified, a *Pass - Fail* quality check is created for that specific work order, rather than the
+manufacturing order as a whole.
+
+*Pass - Fail* quality checks configured for work orders must be processed from the tablet view. To
+do so, begin by navigating to :menuselection:`Manufacturing --> Operations -->
+Manufacturing Orders`. Select a manufacturing order that includes a work order for which a quality
+check is required. Open the tablet view for that work order by selecting the :guilabel:`Work Orders`
+tab, and then clicking the :guilabel:`📱 (tablet)` button on the order's line.
+
+With tablet view open, complete the steps listed on the left side of the screen until the *Pass -
+Fail* quality check step is reached. Upon reaching the check, follow the instructions that appear at
+the top of the screen. If the criteria for the check is met, click the :guilabel:`Pass` button at
+the top-right of the screen. If the criteria is not met, click the :guilabel:`Fail` button.
+
+.. image:: pass_fail_check/work-order-pass-fail-check.png
+   :align: center
+   :alt: A Pass - Fail check for a manufacturing work order.
```

Comment:
Yay! I appreciate that you applied my suggestion for a previous PR in this one too :smile: